### DISCUSSION TITLE
PLEASE HELP IDK WHAT TO DO

### DISCUSSION THREAD

**Discussion by Slorpion** [Pryaxis/TShock#2099] [General]
![Capture g](https://user-images.githubusercontent.com/68771986/88453026-5b07db80-ce18-11ea-8e82-1749851489c1.PNG)

**Comment by Monoverde888** [comment]
@hakusaro

**Reply by hakusaro** [reply]
@Monoverde888 no, it’s not okay to do this

**Comment by tylerjwatson** [comment]
@Monoverde888 Please refrain from tagging Pryaxis staff members in discussion threads.  Maintaining open-source projects is a big effort, behavior like this just cloggs up support and is not appreciated.

**Comment by Monoverde888** [comment]
Sorry

El jue., 10 de sep. de 2020 a la(s) 20:22, Tyler Watson (
notifications@github.com) escribió:

> @Monoverde888 <https://github.com/Monoverde888> Please refrain from
> tagging Pryaxis staff members in discussion threads. Maintaining
> open-source projects is a big effort, behavior like this just cloggs up
> support and is not appreciated.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Pryaxis/TShock/discussions/2099#discussioncomment-76578>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKLICGZDNN4EZZICPH4XTZLSFFNTNANCNFSM4PHLK7RQ>
> .
>


--
